stable/insights-agent/templates/nova/cronjob.yaml

```diff
@@ -27,7 +27,7 @@ spec:
             command:
               - /nova
               - find
-              - --config=/config/nova.yaml
+              - --config={{ .Config.configLocation }}
```

Comment:
`.Values`?

Reply:
According to line three of that template, `.Config` is set to `.Values.nova`

Reply:
so no?

Reply:
Oh thank you. I'm even the one that set that up 🤦